src/lib/slack/index.js

```diff
@@ -6,28 +6,29 @@ const router = require('express').Router(),
 
 // Dumb hack to deal with Lambda's premature execution termination
 async function route(res, promise) {
-    if (process.env.EXECUTION_CONTEXT != 'LAMBDA') {
+    if (process.env.EXECUTION_CONTEXT !== 'LAMBDA') {
         res.status(200).send();
     }
 
-    await promise
-    .catch((err) => {
+    try {
+        await promise
+    } catch (err) {
         console.error('ERROR ** Caught unhandled error in API execution:', err);
-    });
+    }
 
-    if (process.env.EXECUTION_CONTEXT == 'LAMBDA') {
+    if (process.env.EXECUTION_CONTEXT === 'LAMBDA') {
```

Comment:
Same question on triple equals. Nothing type-converts to a particular string.

Reply:
Same as above as well